sql/core/src/test/scala/org/apache/spark/sql/SQLQuerySuite.scala

```diff
@@ -2513,4 +2513,18 @@ class SQLQuerySuite extends QueryTest with SharedSQLContext {
       }
     }
   }
+
+  test(
+    "SPARK-19059: Unable to retrieve data from parquet table whose name startswith underscore") {
```

Comment:
this bug is not parquet only right? how about `SPARK-19059: read file based table whose name starts with underscore`